deployments/deployment.yaml

```diff
@@ -576,7 +577,7 @@ parameters:
   - name: IMAGE
     value: quay.io/cloudservices/content-sources-backend
   - name: NIGHTLY_CRON_JOB
-    value: "0 0/8 * * *"
+    value: "0 0/24 * * *"
```

Comment:
```suggestion
    value: "0 0/1 * * *"
```
I think it would need to be 1 to run once an hour, right?

Reply:
hah, good call!